x-pack/plugins/actions/server/builtin_action_types/swimlane/schema.ts

```diff
@@ -0,0 +1,67 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import { schema } from '@kbn/config-schema';
+
+export const ConfigMap = {
+  id: schema.string(),
+  key: schema.string(),
+};
+
+export const ConfigMapSchema = schema.object(ConfigMap);
+
+export const ConfigMapping = {
+  alertSourceKeyName: ConfigMapSchema,
+  severityKeyName: ConfigMapSchema,
+  caseNameKeyName: schema.nullable(ConfigMapSchema),
```

Comment:
Hey! Would mind explaining me logic of those variables (`caseNameKeyName`, `caseIdKeyName`) ? Are they related to Elastic's Cases?